File: src/net/eduard/economy/command/EconomyShowCommand.java

package net.eduard.economy.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.api.lib.modules.Extra;
import net.eduard.api.lib.player.FakePlayer;
import net.eduard.economy.EduEconomy;

public class EconomyShowCommand extends CommandManager {

	public EconomyShowCommand() {
		super("show", "ver", "checar", "check");
		setDescription("Ver o dinheiro de outro jogador");
		setUsage("/economy ver <jogador>");

	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		if (args.length == 0) {
			sendUsage(sender);
		} else {
			String jogador = args[0];
			FakePlayer fake = new FakePlayer(jogador);

			sender.sendMessage("§aO dinheiro do jogador " + jogador + " é "
					+ Extra.formatMoney(EduEconomy.getInstance().getManager().getCoins(fake)));

		}
		return true;
	}

}

File: src/main/java/net/eduard/economy/command/EconomyShowCommand.kt
package net.eduard.economy.command

import net.eduard.api.lib.manager.CommandManager
import net.eduard.api.lib.modules.FakePlayer
import net.eduard.api.lib.modules.Extra
import net.eduard.economy.EduEconomyPlugin
import org.bukkit.command.CommandSender

class EconomyShowCommand : CommandManager("show", "ver", "checar", "check") {
    override fun command(sender: CommandSender, args: Array<String>) {
        val instance = EduEconomyPlugin.instance
        if (args.isEmpty()) {
            sendUsage(sender)
            return
        }
        val playerName = args[0]
        val fakePlayer = FakePlayer(playerName)
        val quantidade = Extra.formatMoney(
            instance.manager.getCoins(fakePlayer)
        )
        sender.sendMessage(instance
                .message("money-player-check")
            .replace("%amount", quantidade)
            .replace("%player", playerName)
        )

    }


    init {
        description = "Ver o dinheiro de outro jogador"
        usage = "/money ver <jogador>"
    }
}